src/cursor-doc/model.ts

```diff
@@ -9,6 +9,17 @@ export function initScanner(maxLength: number) {
   scanner = new Scanner(maxLength);
 }
 
+/**
+ * @param text The text to look at for the line ending seq.
+ * @returns The first line ending sequence found in TEXT, if any.
+ */
+export function getFirstLineEndingSeq(text: string) {
```

Comment:
`LineEndingSeq` is called `eol` elsewhere in the codebase. Maybe `getFirstEol` is a good name for this function? Or just `firstEol`.

Reply:
I've changed all occurances of LineEndingSeq to Eol